examples/tenant.yaml

```diff
@@ -181,6 +181,15 @@ spec:
   #   value: "off" # to turn-off browser
   # - name: MINIO_STORAGE_CLASS_STANDARD
   #   value: "EC:2"
+  #   ## For secure env vars like passwords, create an opaque Kubernetes secret and specify the secret in 
+  #   ## the `valueFrom` field. The `valueFrom` object must contain the following fields:
+  #   ## - `name` - the secret from which MinIO extracts the password
+  #   ## - `key` - the data field within secret, whose value will be set to the env variable's value
+  # - name: MINIO_IDENTITY_LDAP_LOOKUP_BIND_PASSWORD
+  #   valueFrom:
+  #     secretKeyRef:
```

Comment:
The explanation above implies that this is a static field that does not change. Is that correct?

Reply:
`secretKeyRef` is a top level field that contains `name` and `key`. `key` has to match the key present in the secret itself. If there is a match, it's value will be fetched and used as the value of this env variable

Reply:
In that case maybe we can say "The `valueFrom` object contains an object `secretKeyRef` which must contain the following fields:" on line 185. It's not super necessary because I think the example makes it clear enough for most users.

Reply:
As per convo in slack, resolving this thread.